packages/integral_isolates/lib/src/isolate_configuration.dart

```diff
@@ -29,8 +29,21 @@ class IsolateConfiguration<Q, R> {
     );
   }
 
+  IsolateConfiguration<Q, R> copyWith({required Q message}) {
```

Comment:
Is `message` the only argument that should be possible to change with `copyWith`?

Reply:
Since message is a named argument I think it is fine to keep it as `copyWith`, just wanted to double check that you hadn't missed any other intended arguments.